gnome-initial-setup/eos/hack-sound-client/eos-hack-sound-client.c

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2019 Endless Mobile. Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * Written by:
+ *     Cesar Fabian Orccon Chipana <cfoch.fabian@gmail.com>
+ */
+
+#include "eos-hack-sound-client.h"
+
+static void eos_hack_sound_client_initable_iface_init (GInitableIface *iface);
+static void eos_hack_sound_client_finalize            (GObject *object);
+static gboolean eos_hack_sound_client_initable_init (GInitable    *initable,
+                                                     GCancellable *cancellable,
+                                                     GError      **error);
+static void play_sound_cb (GDBusProxy   *proxy,
+                           GAsyncResult *res,
+                           GTask        *task);
+static void stop_sound_cb (GDBusProxy   *proxy,
+                           GAsyncResult *res,
+                           GTask        *task);
+
+typedef struct _EOSHackSoundClientPrivate EOSHackSoundClientPrivate;
+
+struct _EOSHackSoundClient
+{
+  GObject parent_instance;
+  EOSHackSoundClientPrivate *priv;
+};
+
+struct _EOSHackSoundClientPrivate {
+  GDBusProxy *proxy;
+};
+
+#define eos_hack_sound_client_parent_class parent_class
+#define _do_init \
+  G_ADD_PRIVATE (EOSHackSoundClient); \
+  G_IMPLEMENT_INTERFACE (G_TYPE_INITABLE, eos_hack_sound_client_initable_iface_init);
+
+#define DEFAULT_PROXY_FLAGS (G_DBUS_PROXY_FLAGS_DO_NOT_LOAD_PROPERTIES | G_DBUS_PROXY_FLAGS_DO_NOT_CONNECT_SIGNALS)
```

Comment:
I don't think this `#define` helps readability – it just moves the flags 150 lines of code away from where they're used.